webdriver/tests/print/printcmd.py

```diff
@@ -13,8 +15,8 @@ def do_print(session, options):
 
 
 def assert_pdf(data):
-    assert data.startswith("%PDF-"), "Decoded data starts with the PDF signature"
-    assert data.endswith("%%EOF\n"), "Decoded data ends with the EOF flag"
+    assert data.startswith(b"%PDF-"), b"Decoded data starts with the PDF signature"
```

Comment:
I don't think the second argument to `assert` needs to be a byte string?

Reply:
Updated. Thank you!